### ISSUE TITLE
After Update all previous backups gone !

### ISSUE THREAD

**Issue by CamFlyerCH** [elpatron68/SnowRunner-Tool#8]
I just updated to 1.0.5.0 and all my previous Backups are gone ! Is this because the backup folder is in a different place (Documents\SRToolBackup\epic instead of Documents\My Games\SnowRunner\SRToolBackup) or was the deletion part of the update to Season 8 ?

**Comment by CamFlyerCH** [comment]
I just checked my backup, and it was definitely the update that killed the files instead of moving them. And they are not in the trashcan.

**Comment by CamFlyerCH** [comment]
I restored the save from backup, restarted the tool and now the files are moved, but the timestamps are wrong. 
Could it be that wenn launched from the setup program the move failed ?
The timestamp shown is the creation timestamp and not the last modified timestamp. The creation timestamp is not restored by the Windows backup software, only the modified date.

**Comment by elpatron68** [comment]
Hi. Thanks for reporting your issue!

The tool uses new directories for it´s backups to be able to distinguish between Epic- and Steam- platforms.

Before 1.0.5 (A): `c:\users\<username>\Documents\My Games\SnowRunner\SRToolBackup`, later (B) `c:\users\<username>\Documents\SRToolBackup`.

At the start, it looks for existent files in (A) and moves them to (B).

I tried to reproduce your issue, but it worked as expected. I will change the move operation to copy -> delete (if successful) with the next update.

**Comment by CamFlyerCH** [comment]
I don't know how, but can delete be done in a way, so the deleted files and folders appear in the trashcan instead of being gone totally ? This would also be good while deleting backups manually from the list. Perhaps this helps: 
[DeleteFile(String, UIOption, RecycleOption](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.fileio.filesystem.deletefile?redirectedfrom=MSDN&view=net-6.0#microsoft-visualbasic-fileio-filesystem-deletefile(system-string-microsoft-visualbasic-fileio-uioption-microsoft-visualbasic-fileio-recycleoption))

**Comment by elpatron68** [comment]
Good point, will be implemented with the next update, at least for deleting backups from the UI. Unfortunately C# does not support deleting to recycle bin (your example uses `Microsoft.VisualBasic.FileIO`), but I have found an alternative implementation which works fine.

The initial problem was caused by an attempt to move the old backup files to the new location at a time where the new directory wasn´t created yet. That is fixed and should not cause any problems in the future.
